api/src/routes/user.ts

```diff
@@ -49,5 +49,69 @@ export const userRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.post(
+    '/account/reset-progress',
+    {
+      schema: {
+        response: {
+          200: Type.Object({}),
+          500: Type.Object({
+            message: Type.Literal(
+              'Oops! Something went wrong. Please try again in a moment or contact support@freecodecamp.org if the error persists.'
+            ),
+            type: Type.Literal('danger')
+          })
+        }
+      }
+    },
+    async (req, reply) => {
+      try {
+        await fastify.prisma.userToken.deleteMany({
+          where: { userId: req.session.user.id }
+        });
+        await fastify.prisma.user.update({
+          where: { id: req.session.user.id },
+          data: {
+            progressTimestamps: [Date.now()],
+            currentChallengeId: '',
+            isRespWebDesignCert: false,
+            is2018DataVisCert: false,
+            isFrontEndLibsCert: false,
+            isJsAlgoDataStructCert: false,
+            isApisMicroservicesCert: false,
+            isInfosecQaCert: false,
+            isQaCertV7: false,
+            isInfosecCertV7: false,
+            is2018FullStackCert: false,
+            isFrontEndCert: false,
+            isBackEndCert: false,
+            isDataVisCert: false,
+            isFullStackCert: false,
+            isSciCompPyCertV7: false,
+            isDataAnalysisPyCertV7: false,
+            isMachineLearningPyCertV7: false,
+            isRelationalDatabaseCertV8: false,
+            // TODO: check if schema should be updated with below field
+            // isCollegeAlgebraPyCertV8: false
```

Comment:
```suggestion
            isCollegeAlgebraPyCertV8: false
```
Yep, that can be set now.